Sources/Subscription/StoreSubscriptionConfiguration.swift

```diff
@@ -19,6 +19,12 @@
 import Foundation
 import Combine
 
+/// Constants relating to `StoreSubscriptionConfiguration`
+enum StoreSubscriptionConstants {
+    /// The suffix appended to subscription identifiers to indicate that the subscription includes a free trial.
+    static let freeTrialSuffix = ".freetrial"
+}
```

Comment:
Minor comment to that:
1. I would drop the `.` from the value as it is semantic connector for identifier
2. It is called a suffix but technically not used as one:
 - in l:56-57 `.dev` is appended
 - elsewhere is checked with `id.contains(StoreSubscriptionConstants.freeTrialSuffix)`

Reply:
Agree, thanks @miasma13. Have updated based on feedback.